### ISSUE TITLE
GOG Version PAK files don't work with Dhewm3

### ISSUE THREAD

**Issue by HardDiskDog** [dhewm/dhewm3#592]
I figured out that the solution to my issue [here](https://github.com/dhewm/dhewm3/issues/590) was to use the Steam version pak files instead of the GOG version ones to make the Dhewm3 executable actually launch, so it would very much appear that Dhewm3 lacks compatibility with the GOG version.

Luckily I happen to own both versions, but people who only own Doom 3 on GOG would be unable to use this source port until an update to address this issue is released. Unless there's something I'm missing, I'm baffled I've never seen someone talk about running into this before. Do that few people buy the GOG version so they don't have to redo compatibility modding every time they reinstall this old game on modern Windows?

**Comment by DanielGibson** [comment]
Are you sure you didn't accidentally try to use the Doom3 **BFG** gamedata from GOG?

**Comment by HardDiskDog** [comment]
Absolutely nothing worked to get Dhewm3 to launch with the GOG pak files, but changing nothing about my install except swapping in the Steam pak files completely fixed it.

I am absolutely sure I've never once downloaded the BFG Edition; I make sure not to since I really don't like it.

In my experience, sometimes GOG versions have slight differences that aren't very documented.

**Comment by DanielGibson** [comment]
Do the .pk4 files listed here: https://dhewm3.org/#getting-the-doom3-game-data
Exist in the GOG version? what are they MD5 sums?
Does dhewm3 show any error messages when trying to start it with the GOG data?

**Comment by HardDiskDog** [comment]
> Do the .pk4 files listed here: https://dhewm3.org/#getting-the-doom3-game-data Exist in the GOG version? what are they MD5 sums? Does dhewm3 show any error messages when trying to start it with the GOG data?

As I said in the original post I linked, once all the GOG pak files are in place, double-clicking dhewm3.exe just crashed before even starting up without an error message.
I would get those missing .dll error messages I listed, for files that don't come with with either the GOG or Steam versions, (and don't seem required at all to run the game, unless they're Windows system folders or something) seemingly as a result of having not yet placed the pak files. The latter scenario is obviously not how to play, but might provide some insight into the prior one.

**Comment by DanielGibson** [comment]
Usually dhewm3 would show a window with error messages when it can't find the right game data.

Does any log get written in `Documents\My Games\dhewm3\base\`?

You could try to compare which files exist in your doom3+dhewm3 directory with the GOG version vs the steam version and figure out which are different (file only exists in one place, or file has different size or checksum (MD5 sum, sha256sum or whatever kind you prefer)).

**Comment by HardDiskDog** [comment]
Strangely, downloading a fresh Dhewm3 folder from the website and placing in the pak files from the GOG version allowed it to launch without issues.
Maybe it was something with my previous GOG version installation, since I installed a separate new one from GOG to copy the PAK files from for a clean slate, but I already tried a fresh Dhewm3 folder with nothing transferred from GOG except the pak files (making a Steam vs GOG comparison seem irrelevant) and was still having the issue. Very strange.
Log files from when I had the startup issue unfortunately seem to have since been overwritten.
